src/test/java/edu/ucsb/mapache/controllers/SlackSlashCommandControllerTests.java

```diff
@@ -21,405 +19,251 @@
-    mockMvc
-        .perform(post(testURL).contentType(MediaType.APPLICATION_FORM_URLENCODED_VALUE)
-        .param("token", slackSlashCommandController.getSlackToken())
-        .param("team_id", "value")
-        .param("team_domain", "value")
-        .param("channel_id", "value")
-        .param("channel_name", "value")
-        .param("user_id", "value")
-        .param("user_name", "value")
-        .param("command", "/mapache")
-        .param("text", "search google two words")
-        .param("response_url", "value")
-        )
-        .andExpect(status().is(200));
-  }
-
-  @Test
-  public void test_googleSearch_empty_items() throws Exception {
+        Path jsonPath = Paths.get("src/test/java/edu/ucsb/mapache/google/sample.json");
+        String retval = Files.readString(jsonPath);
+        when(googleSearchServiceHelper.getJSON(any(), any())).thenReturn(retval);
+        mockMvc.perform(post(testURL).contentType(MediaType.APPLICATION_FORM_URLENCODED_VALUE)
+                .param("token", slackSlashCommandController.getSlackToken()).param("team_id", "value")
+                .param("team_domain", "value").param("channel_id", "value").param("channel_name", "value")
+                .param("user_id", "value").param("user_name", "value").param("command", "/mapache")
+                .param("text", "search google two words").param("response_url", "value")).andExpect(status().is(200));
+    }
+
+    @Test
+    public void test_googleSearch_empty_items() throws Exception {
         // content type: https://api.slack.com/interactivity/slash-commands
-    Path jsonPath = Paths.get("src/test/java/edu/ucsb/mapache/google/sample.json");
-    String retval = Files.readString(jsonPath);
-    int index_trim = retval.indexOf("\"items\"");
-    retval = retval.substring(0, index_trim) + "\"items\": []}";
-    when(googleSearchService.getJSON(any(), any())).thenReturn(retval);
-    SlackSlashCommandParams params = new SlackSlashCommandParams();
-    params.setText("search google test");
-    RichMessage result = slackSlashCommandController.googleSearch(params);
-    assertEquals("No results found!", result.getAttachments()[0].getText());
-  }
-  // WIP unable to test IOexception currently
-  @Test
-  public void test_googleSearch_1() throws Exception {
+        Path jsonPath = Paths.get("src/test/java/edu/ucsb/mapache/google/sample.json");
+        String retval = Files.readString(jsonPath);
+        int index_trim = retval.indexOf("\"items\"");
+        retval = retval.substring(0, index_trim) + "\"items\": []}";
+        when(googleSearchServiceHelper.getJSON(any(), any())).thenReturn(retval);
+        SlackSlashCommandParams params = new SlackSlashCommandParams();
+        params.setText("search google test");
+        RichMessage result = slackSlashCommandController.googleSearch(params);
+        assertEquals("No results found!", result.getAttachments()[0].getText());
+    }
+
+    // WIP unable to test IOexception currently
```

Comment:
Is this still the case?

Reply:
I don't think so.  So I removed that comment.